kyo-prelude/shared/src/main/scala/kyo2/Result.scala

```diff
@@ -77,23 +86,26 @@ object Result:
             self.isInstanceOf[Panic]
 
         inline def fold[B](inline ifError: E => B)(inline ifPanic: Throwable => B)(inline ifSuccess: A => B): B =
```

Comment:
Reduces the inlined bytecode size by moving some of the logic to `get`. It'll now throw in case of `Panic` but, since it won't produce a new stack trace, there isn't a performance penalty.